### ISSUE TITLE
Did you try driving a video?

### ISSUE THREAD

**Issue by Souvic** [StelaBou/StyleMask#6]
Did you try driving a video? How did it go?

**Comment by StelaBou** [comment]
Yes it is possible and the result is quite good. The only problem with videos is that due to the StyleGAN2 model (trained on FFHQ) that I am using, the range of facial expressions is limited to the distribution of FFHQ dataset. As a result in case of real talking head videos, the reenacted images may not be able to capture the full range of the facial expressions.
